.github/workflows/R-CMD-check.yaml

```diff
@@ -18,29 +18,48 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          # - {os: macOS-latest,   r: 'release'}
-          - {os: windows-latest, r: 'release'}
-          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
-          - {os: ubuntu-latest,   r: 'release'}
-          - {os: ubuntu-latest,   r: 'oldrel-1'}
-
+          - {os: macOS-latest,   r: 'release'}
+          #- {os: windows-latest, r: 'release'}
+          - {os: ubuntu-20.04,   r: 'devel'}
+          #- {os: ubuntu-latest,   r: 'release'}
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_KEEP_PKG_SOURCE: yes
 
     steps:
       - uses: actions/checkout@v2
+      
+      - uses: r-lib/actions/setup-r@v2
 
+      - name: Install macOS system dependencies
+        if: runner.os == 'macOS'
+        run: |
+          env HOMEBREW_NO_AUTO_UPDATE=1 brew install hdf5 proj gsl
+     
+      - uses: r-lib/actions/setup-r-dependencies@v2
```

Comment:
Can this block be moved after "Install Linux system dependencies", so that the whole flow is:

- checkout@v2
- setup-r@v2
- setup-pandoc@v1
- Install MacOS system dep
- Install Linux system dep
- setup-r-dependencies@v2
- Install dependencies
- check-r-package@v2

Reply:
I ran into two issues earlier:
- The Mac systems dependencies need to be installed _before_ calling setup-r-dependencies@v2, since vol2birdR is a dependencies and requires those system libraries.
- The install Linux system dep block uses package `pak`, which is install with setup-r-dependencies@v2 and has to come after that.

So in your above list you have to swap "Install Linux system dep" and "setup-r-dependencies@v2".

Reply:
I'm not sure if setup-pandoc@v1 can be moved up

Reply:
moving pandoc up works, resolving this one